src/Providers/FindologicServiceProvider.php

```diff
@@ -52,37 +63,47 @@ public function boot(
             return;
         }
 
+        $this->isSearchPage = strpos($request->getUri(), '/search') !== false;
+        $this->activeOnCatPage = !$this->isSearchPage && $configRepository->get(
```

Comment:
Remove the default value for `CONFIG_NAVIGATION_ENABLED` since it can never be empty.